roles/rojinebrahimi.wordpress/vars/main.yml.save

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
what does this file do?

Reply:
I have to remove it. That does not do anything.